dvc/cachemgr.py

```diff
@@ -73,6 +74,16 @@ def __init__(self, repo):
         legacy_odb = _get_odb(repo, settings, hash_name="md5-dos2unix", **kwargs)
         self._odb["legacy"] = legacy_odb
 
+    @property
+    def fs(self):
```

Comment:
This will be confusing with `fs` the filesystem. Maybe `file_storage` or `file_store`?

Reply:
Also thought about that, but it is a property of cache manager, not particular cache. It is meant to mean `fs_storage` or `fs_cache`, since it is a per-fs cache. E.g. we have `dvc.cache.local`, `dvc.cache.repo`, `dvc.cache.s3`(obsoleted) and `dvc.cache.fs`. I'll rename it to `fs_cache` then.